central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -80,13 +80,15 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 		operatorErrors = append(operatorErrors, fmt.Sprintf("Compliance operator is not ready. Only %d pods out of desired %d are ready.", readyPods, desiredPods))
 	}
 
-	// we support only newer versions of compliance operator
-	complianceOperatorVersion, err := semver.NewVersion(operatorInfo.GetVersion())
-	if complianceOperatorVersion == nil || err != nil {
-		log.Error(errors.Wrapf(err, "parsing compliance operator version: %q", operatorInfo.GetVersion()))
-		operatorErrors = append(operatorErrors, fmt.Sprintf("The installed compliance operator version %q is invalid.", operatorInfo.GetVersion()))
-	} else if complianceOperatorVersion.LessThan(env.ComplianceMinimalSupportedVersion.VersionSetting()) {
-		operatorErrors = append(operatorErrors, fmt.Sprintf("The installed compliance operator version %q is unsupported. The minimum required version is %q.", complianceOperatorVersion.String(), env.ComplianceMinimalSupportedVersion.VersionSetting().String()))
+	if msg.GetComplianceOperatorInfo().GetIsInstalled() {
```

Comment:
That looks great @mtodor ! 

I am just asking myself whether we should maybe do

```go
if !msg.GetComplianceOperatorInfo().GetIsInstalled() {
    return nil
}
```

somewhere around line 62. WDYT?

Reply:
Note line 96:

```
	if len(operatorInfo.StatusErrors) == 0 {
		operatorInfo.OperatorStatus = storage.COStatus_HEALTHY
	}
```

Wouldn't we mark this cluster as healthy if `IsInstalled` is false?

(this is actually not related to this PR, so feel free to resolve)

Reply:
If the operator is not installed we should return a `NotFound` error in `StatusError`.

Reply:
@mtodor , where the following is addressed?
> If the operator is not installed we should return a NotFound error in  StatusError.